internal/server/data/data.go

```diff
@@ -23,16 +23,20 @@ import (
 	"github.com/infrahq/infra/uid"
 )
 
-type NewDBOptions struct {
+type Options struct {
```

Comment:
`NewDBOptions` is somewhat more explicit. There are a lot of functions in the `data` package, and many structs named `<name of operation>Options`. Doesn't calling this `Options` make it less obvious how to use it?